src/app/components/datepicker/index.html

```diff
@@ -5,71 +5,75 @@
 >
 
   <sky-demo-page-summary>
-    The datepicker module allows users to use an input and calendar to select dates.
+    The datepicker module contains directives to let users select dates.  It is a SKY UX-themed replacement for the HTML <stache-code>input</stache-code> element with <stache-code>type="date"</stache-code>. The value that users select is driven through the <stache-code>ngModel</stache-code> attribute
+    specified on the <stache-code>input</stache-code> element.
```

Comment:
Is there a reason for the line break here?

Reply:
Nope. I'll remove the line break.